coalib/bears/Bear.py

```diff
@@ -24,6 +24,26 @@
 from .meta import bearclass
 
 
+def _debugger_setting_value(bear):
```

Comment:
Would change the name, it's not descriptive enough.
So what does this function do? You want to check whether the bear is in debug mode according to its section-settings. So a name like `_is_debugged` or `_is_debug_mode_enabled` or similar might fit better^^